.changeset/blue-poets-pull.md

```diff
@@ -0,0 +1,5 @@
+---
+"@razorpay/blade": min
```

Comment:
Should be patch?
```suggestion
"@razorpay/blade": patch
```

Reply:
yeah confused. because it was not broken and added as a functionality thought it might be minor?